hypertrace-core-graphql-service/src/main/java/org/hypertrace/core/graphql/service/DefaultGraphQlServiceConfig.java

```diff
@@ -111,6 +120,14 @@ public Duration getGatewayServiceTimeout() {
     return gatewayServiceTimeout;
   }
 
+  private Duration getTimeoutOrFallback(Supplier<Duration> durationSupplier) {
```

Comment:
nit - this is just another wrapper on 
```java
optionallyGet(supplier)
  .orElse(Duration.ofSeconds(DEFAULT_CLIENT_TIMEOUT_SECONDS));
```
If you'd prefer to share the orElse in a common method like this that's fine, but I'd encourage it to still use optionallyGet underneath so we're not duplicating the missing behavior twice

Reply:
Yes that method was kind-of redundant @aaron-steinfeld . Addressed.